modules/nlu/src/backend/module-lifecycle/on-server-started.ts

```diff
@@ -62,7 +62,7 @@ function initializeEngine(bp: typeof sdk, state: NLUState) {
     reportTrainingProgress: async (botId: string, message: string, trainSession: TrainingSession) => {
       await setTrainingSession(bp, botId, trainSession)
 
-      const ev = {
+      const ev: NluProgressEvent = {
```

Comment:
we try to keep Nlu in caps all around the codebase as this is an acronym. 

```suggestion
      const ev: NLUProgressEvent = {
```